esl1350/epitaph
# Make Blink Dagger do AoE dmg on reactivation

Modified Blink Dagger Ability to deal AoE damage at the teleport destination :)
AoE radius and damage are modifiable in-editor

hopefully better now 💀

## Description


## Related Task

[Task](https://app.clickup.com/t/8678rtgr4)

## Testing
I made a testing scene with a Goon that can't move, and checked that the radius and damage worked.

- Hit an enemy to see if he loses HP
- Hit walls and non-entities to check the ability doesn't bother them
- Try near misses to double check colliders are behaving
- Check the player doesn't lose HP when you teleport

Changes limited to BlinkDaggerAbility.cs and BlinkDaggerProjectile.cs, so shouldn't affect anything else





## Screenshots (if appropriate):
